examples/run-app.sh

```diff
@@ -52,10 +57,20 @@ fi
 # Export the Data broker DAPR_GRPC_PORT
 export DAPR_GRPC_PORT=55555
 
-DAPR_APP_ID="${APP_NAME//[.,\-,_,' ']/}"
-dapr run \
-    --app-id $DAPR_APP_ID \
-    --app-protocol grpc \
-    --app-port $APP_PORT \
-    --config $ROOT_DIRECTORY/.dapr/config.yaml  \
-    --components-path $ROOT_DIRECTORY/.dapr/components python3 $APP_NAME/src/main.py
+if [ $SDV_MIDDLEWARE_TYPE == "dapr" ]; then
+    echo "Run with Dapr ...!"
+    DAPR_APP_ID="${APP_NAME//[.,\-,_,' ']/}"
+    dapr run \
+      --app-id $DAPR_APP_ID \
+      --app-protocol grpc \
+      --app-port $APP_PORT \
+      --config $ROOT_DIRECTORY/.dapr/config.yaml  \
+      --resources-path $ROOT_DIRECTORY/.dapr/components \
+    -- python3 $APP_NAME/src/main.py
+elif [ $SDV_MIDDLEWARE_TYPE == "native" ]; then
+    echo "Run native ...!"
+    python3 $APP_NAME/src/main.py
```

Comment:
When running in native mode, the native middleware service locator will expect the full address of services to be available as env variables, see here: https://github.com/eclipse-velocitas/vehicle-app-python-sdk/blob/main/sdv/native/locator.py#L34

Hence we have to export some env vars before invoking the python script directly

Reply:
Fixed in new PR: https://github.com/eclipse-velocitas/vehicle-app-python-sdk/pull/73